io.github.obsidian-level-maker.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.obsidian-level-maker
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --filesystem=home:rw
```

Comment:
this is a broad permission.

Reply:
That is a subject term. Please explain what you mean.

Reply:
that granting permission to the whole HOME directory is considered broad.

Reply:
This program generates maps for a variety of Doom engine games. We cannot predict how a user organizes the custom content for their games nor where they are stored within the user's home directory.